meta-delta/conf/layer.conf

```diff
@@ -0,0 +1,15 @@
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+	${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "delta"
+BBFILE_PATTERN_facebook = "^${LAYERDIR_RE}/recipes"
+BBFILE_PRIORITY_facebook = "15"
+
+
+LAYERSERIES_COMPAT_facebook = "gatesgarth dunfell zeus warrior rocko"
```

Comment:
Can this just be Dunfell and above and not including  Zeus warrior or rocko?

Reply:
I would remove warrior and rocko. But since Delta currently is using Zeus with our local changes, I think I would need to keep Zeus and above for now. Other versions will be removed if Dunfell is testing OK on our platform in futures.